### ISSUE TITLE
Rules timer bug.

### ISSUE THREAD

**Issue by Budman1758** [enesbcs/rpieasy#31]
I think I have found another minor bug in the rules engine.  When you have a timer running and want to cancel it for whatever reason the command is is to set the timer to 0.  (timerset,1,0)
What happens is the command is recognized but the result in the log is Event: Rules#timer=1.  This does not cancel the rule.  It has the effect of just shortening it because what ever is supposed to happen when the timer expires still does.  

In short, the timer is not actually cancelled. Whatever logic is defined to occur when it expires naturally is still executed.

**Comment by enesbcs** [comment]
yes, as i see timerset,1,0 does not cancels the timer, instead trigger it's event.

**Comment by enesbcs** [comment]
I think i fixed it in commit https://github.com/enesbcs/rpieasy/commit/cd9eadc5a5a19134dd4acccef784940de11e26d8 Please try it.

**Comment by Budman1758** [comment]
You must be an Exterminator in your day job.  You're pretty fast at killing those bugs.  :>)

**Comment by enesbcs** [comment]
In cases when i have some ideas, yes!  :)
